src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -0,0 +1,460 @@
+  const withdraw = availableShortcutIds.includes('withdraw')
+
+  return (
+    <View style={[styles.buttonContainer, insetsStyle]}>
+      {withdraw && (
+        <Button
+          text={t('earnFlow.poolInfoScreen.withdraw')}
+          onPress={() => {
+            // TODO (ACT-1343): EarnCollectScreen should take earnPositon instead of depositTokenId and poolTokenId
+            // navigate(Screens.EarnCollectScreen, { earnPosition })
+            Logger.debug('Withdraw Button Pressed!')
+          }}
+          size={BtnSizes.FULL}
+          type={BtnTypes.SECONDARY}
+          style={styles.flex}
+        />
+      )}
+      {deposit && (
+        <Button
+          text={t('earnFlow.poolInfoScreen.deposit')}
+          onPress={() => {
+            // TODO hook up after ACT-1342 is merged
+            // navigate(Screens.EarnEnterAmount, { pool: earnPosition })
+            Logger.debug('Deposit Button Pressed!')
+          }}
+          size={BtnSizes.FULL}
+          style={styles.flex}
+        />
+      )}
+    </View>
+  )
+}
+
+type Props = NativeStackScreenProps<StackParamList, Screens.EarnPoolInfoScreen>
+
+export default function EarnPoolInfoScreen({ route, navigation }: Props) {
+  const { pool } = route.params
+  const { networkId, tokens, displayProps, appName, dataProps } = pool
+  const allTokens = useSelector((state) => tokensByIdSelector(state, [networkId]))
+  const tokensInfo = useMemo(() => {
+    return tokens
+      .map((token) => allTokens[token.tokenId])
+      .filter((token): token is TokenBalance => !!token)
+  }, [tokens, allTokens])
+
+  // console.log('Tom - earnPosition.dataProps.yieldRates: ', pool.dataProps.yieldRates)
+
+  // Scroll Aware Header
+  const scrollPosition = useSharedValue(0)
+  const [titleHeight, setTitleHeight] = useState(0)
+  const handleMeasureTitleHeight = (event: LayoutChangeEvent) => {
+    setTitleHeight(event.nativeEvent.layout.height)
+  }
+  const handleScroll = useAnimatedScrollHandler((event) => {
+    scrollPosition.value = event.contentOffset.y
+  })
+  useScrollAwareHeader({
+    navigation,
+    title: <HeaderTitleSection earnPosition={pool} tokensInfo={tokensInfo} />,
+    scrollPosition,
+    startFadeInPosition: titleHeight - titleHeight * 0.33,
+    animationDistance: titleHeight * 0.33,
+  })
+
+  return (
+    <SafeAreaView style={styles.flex} edges={[]}>
+      <Animated.ScrollView contentContainerStyle={styles.scrollContainer} onScroll={handleScroll}>
+        <TitleSection
+          title={displayProps.title}
+          tokensInfo={tokensInfo}
+          providerName={appName}
+          networkName={NETWORK_NAMES[networkId]}
+          onLayout={handleMeasureTitleHeight}
+        />
+        <View style={{ height: Spacing.Thick24 }} />
+        <View style={styles.contentContainer}>
+          <YieldCard
+            // TODO(ACT-1323): Create info bottom sheet
+            infoIconPress={() => Logger.debug('YieldCard Info Icon Pressed!')}
```

Comment:
We generally don't have Logger.debug's right? If this is just to make TS happy and avoid empty functions, you can add analytics events

Reply:
I think It's better to include that work in ACT-1323 as it includes creating and hooking up the bottom sheets. These  `Logger.debug` calls are temporary, but we also use them in other places within the wallet.

Reply:
can we leave a note to remove, unless we strictly need them?